pytest.ini

```diff
@@ -1,5 +1,3 @@
 [pytest]
 addopts = --cov . --ds=edx_sga.test_settings
 norecursedirs = .git .tox edx_sga.static edx_sga.locale edx_sga.templates {arch} *.egg
-pep8ignore = */migrations/* ALL
-pep8maxlinelength = 120
```

Comment:
@umarmughal824 Why do we need to remove these in a PR where we are adding django 32 support?

Reply:
I didn't manually delete that. I had run the black command that has deleted and it didn't go through my eyes. Thanks for the catch